README.md

```diff
@@ -9,53 +9,72 @@ Scripting with Microsoft Azure in a breeze.
 # Installation
 Easy to install it via PowerShell gallery:
 
-- **ARM**
-
-```powershell
-PS> Install-Module -Name Arcus.Scripting.ARM
-```
-
-- **Azure API Management**
-
+## Azure Web
+- **Azure API Management** ([docs](https://scripting.arcus-azure.net/Features/powershell/azure-api-management))
 ```powershell
 PS> Install-Module -Name Arcus.Scripting.ApiManagement
 ```
+- **Azure App Service** ([docs](https://scripting.arcus-azure.net/Features/powershell/azure-app-service))
```

Comment:
this is missing the powershell part showing how to install

Reply:
Aha aha, thx for noticing. 🥴